backend/src/main/kotlin/com/group7/artshare/DTO/DiscussionPostDTO.kt

```diff
@@ -11,7 +11,7 @@ class DiscussionPostDTO{
     var posterId : Long? = null
     var creationDate : Date? = null
     var lastEditDate : Date? = null
-    var upvoteNo : Int? = null
-    var downvoteNo : Int? = null
+    var upVotedUserIds : MutableSet<Long> = mutableSetOf()
```

Comment:
it would be better to keep usernames instead of ids

Reply:
Done